components/desktop/i3status/Makefile

```diff
@@ -0,0 +1,46 @@
+#
```

Comment:
Please separate the i3 and i3status into separate PR's.
This is needed because i3 depends on i3status and thus i3status has to be merged and installed before i3 can be merged.